data/config-schema/src/main/resources/net/corda/schema/configuration/cryptoLibrary/1.0/corda.cryptoLibrary.json

```diff
@@ -5,6 +5,41 @@
   "description": "Configuration schema for the crypto library subsection.",
   "type": "object",
   "properties": {
+    "unmanagedWrappingKeys": {
+      "description" : "Key derivation parameters for unmanaged wrapping keys",
+      "type": "object",
+      "default": {},
+      "patternProperties": {
+        ".*": { 
+          "type" :  "object",
+          "properties": {
```

Comment:
It this were to be a list, then we'd need an id property as well.

Reply:
:+1: